Automattic/wp-calypso
# Components: Unify Gridicon styles within LoggedOutFormLinkItem

This PR unifies the `Gridicon` styles within `LoggedOutFormLinkItem`. This combination occurs mostly within the Jetpack Connect flow, once in the disconnection flow and only in one instance in the signup with store flow.

In order to keep everything consistent, the PR adds necessary `size` to `Gridicon` instances where previously we were resizing them with CSS, and adds spaces after icons where necessary to align them all properly.

This PR helps resolve issues like this one: https://github.com/Automattic/wp-calypso/pull/19327#issuecomment-340787245

To test:
* Checkout this branch
* Verify gridicons in footer links are properly aligned vertically and horizontally while testing each step of the following flows:
  * Go through all steps of the /jetpack/connect flow
  * Go through all steps of the /jetpack/connect/store flow
  * Go to /start/store/design-type-with-store and click on "Start with an online store"
  * Go to /settings/disconnect-site/:site
